openslides_backend/action/actions/meeting/create_update_delete.py

```diff
@@ -224,6 +225,24 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
             }
         ]
         self.execute_other_action(UserUpdate, action_data)
+        action_data_countdowns = [
+            {
+                "title": "List of speakers countdown",
+                "meeting_id": instance["id"],
+                "used_as_list_of_speaker_countdown_meeting_id": instance["id"],
+                "default_time": instance["projector_countdown_default_time"],
```

Comment:
Do you need to give the default time? `ProjectorCountdownCreate` should handle it I think. Maybe (@jsangmeister) this is only possible with the DS cache?

Reply:
The meeting does not exist yet. Leads to an error.

Reply:
And it is the same.

Reply:
> this is only possible with the DS cache?

Yes, it should be, but `projector_countdown_default_time` is currently not required and therefore not set on meeting creation (see https://github.com/OpenSlides/openslides-backend/issues/558). I would assume it should work once that issue is resolved.

Reply:
It is a question of order: Are the defaults set earlier than this statement?

Reply:
> Are the defaults set earlier than this statement?

Yes.

> Leads to an error.

You probably have to update the cache (`self.apply_instance(instance)`) before executing the other action. Please implement #558 first, then check here that the default is actually set and write it to the cache and then try again to leave the field out.

Reply:
The apply_instance(instance) set a pre version of the meeting into the cache. Then the default_time is not needed in the action_data_create.